DependencyInjection/ExampleEventExtension.php

```diff
@@ -73,5 +75,52 @@ public function prepend(ContainerBuilder $container)
                 ],
             ]
         );
+
+        $this->prependAdmin($container);
+    }
+
+    private function prependAdmin(ContainerBuilder $container)
+    {
+        if (SuluKernel::CONTEXT_ADMIN !== $container->getParameter('sulu.context')) {
+            return;
+        }
+
+        if (!$container->hasExtension('sulu_admin')) {
+            throw new \RuntimeException('Missing SuluAdminBundle.');
+        }
+
+        $container->prependExtensionConfig(
+            'sulu_admin',
+            [
+                'field_type_options' => [
+                    'assignment' => [
+                        'event_assignment' => [
```

Comment:
Thomas has already pointed out that we should rename that once we have consolidated the naming here, e.g. using https://github.com/sulu/sulu/pull/4038.